modules/administration/pages/hostname-rename.adoc

```diff
@@ -0,0 +1,27 @@
+[[hostname-rename]]
+= Rename {productname} Server
+
+It is possible to rename a {productname} Server.
+You first must change the network settings of the server locally and remotely.
+Then the [command]``spacewalk-hostname-rename`` script will the settings in the {productname} PostgreSQL database and the internal structures of {productname} accordingly.
+
+The [command]``spacewalk-hostname-rename`` script is part of the [package]``spacewalk-utils`` package.
+
+The only mandatory parameter for the script is the newly configured IP address of the {productname} Server.
+
+
+
+.Procedure: Renaming {productname} Server
+. Change the network settings of the server on the system level locally and remotely at the DNS server.
+This part of the procedure is similar to the actions to take if you want to rename any other regular system.
++
+[NOTE]
+====
+You must also provide configuration settings for reverse name resolution.
+====
+. Restart your network configuration to make the {productname} server use the new hostname.
+If the server does not use the new name, the script will not complete.
+After rebooting the server will use the new name.
+. Run the script [command]``spacewalk-hostname-rename`` script with the public IP address of the server.
+. Some other actions are needed to make your landscape aware of the new hostname and IP address.
```

Comment:
Landscape? :thinking: Do you mean environment?

Reply:
I also don't like the vague "some other actions" but that might be the best we can do under the circumstances.

Reply:
I'd give some examples: AFAIK traditional and salt clients MUST be reconfigured as otherwise they will still point to the old hostname. Not sure if "reconfiguring" is the word, or "re-onboarding". Someone with experience should know.

Reply:
I think re-configure is the right word. Think about salt minion config file. You specify

```
master: FQDN.OF.MASTER
```

If you change that hostname, the minion needs to be re-configured.
The traditional client has /etc/sysconfig/rhn/up2date configfile which must be changed.